tests/acceptance/test_gate_seal.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from brownie import Contract, interface, web3  # type: ignore
+from brownie.network.account import Account
+
+from utils.config import (
+    GATE_SEAL_EXPIRY_TIMESTAMP,
+    GATE_SEAL_PAUSE_DURATION_SECONDS,
+    contracts,
+    deployer_eoa,
+    gate_seal_address,
+)
+
+
+@pytest.fixture(scope="module")
+def gate_seal_committee(accounts) -> Account:
```

Comment:
maybe it should be skipped or fail to be more visible

Reply:
Yeah, did it.